azerothcore/azerothcore-wotlk
# feat(Script/Command): Allow GMs to disable getting mail

## Changes Proposed:
- Allow GMs to accept/disallow others from sending mail to them

## SOURCE:


## Tests Performed:

- Built
- Tested .mail (on/off) ingame


## How to Test the Changes:


1. use `.mail (on/off)` and try to send mail to that character
2. Log out and in and see if the state is saved on your character. `.mail` to see current state
3. Change values in config to Disable/Enable/Last save state by default


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
